File: Formula-Bollo-App/src/main/java/formula/bollo/app/entity/Position.java
package formula.bollo.app.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name = "position")
@Getter
@Setter
@ToString
@EqualsAndHashCode
@AllArgsConstructor
@NoArgsConstructor
public class Position {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private long id;

    @Column(name = "position_number", unique = true, nullable = false, length = 6)
    private int positionNumber;

    @Column(name = "points", nullable = false, length = 6)
    private int points;
}

File: Formula-Bollo/src/shared/models/position.ts
import { Result } from "./result";

export class Position {
  id: number;
  positionNumber: number;
  points: number;

  constructor(id: number, positionNumber: number, points: number) {
    this.id = id;
    this.positionNumber = positionNumber;
    this.points = points;
  }
}
